### PR TITLE
[IMP]increace max_conections and max_pred_lock_pre_transaction

### PR DESCRIPTION
**Pull request by hugho-ad**
FIX https://github.com/Vauxoo/yoytec/issues/591

### CODE REVIEW

**Comment by hugho-ad** [comment]
@moylop260 can you review please.

**Comment by moylop260** [comment]
@hugho-ad 
I tested this changes in a small VM and work fine!
screen shot 2016-01-19 at 3 29 45 pm


Just a small comments:
Change the path `/etc/postgresql/9.3/main/postgresql.conf`
by `/etc/postgresql/*/main/postgresql.conf`

File: odoo-shippable/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,8 @@ RUN git config --global user.name Shippable \
 # Install basic postgres
 RUN sudo apt-get update \
     && sudo apt-get install -y postgresql-9.3 postgresql-contrib-9.3
+RUN sed -i 's/#max_pred_locks_per_transaction = 64/max_pred_locks_per_transaction = 100/g' /etc/postgresql/9.3/main/postgresql.conf
```
Comment (moylop260) [inline odoo-shippable/Dockerfile]: s/9.3/*/g

File: odoo-shippable/Dockerfile [hunk 2]
```diff
@@ -22,6 +22,8 @@ RUN git config --global user.name Shippable \
...
+RUN sed -i 's/max_connections = 100/max_connections = 200/g' /etc/postgresql/9.3/main/postgresql.conf
```
Comment (moylop260) [inline odoo-shippable/Dockerfile]: s/9.3/*/g

**Pull request merged**